server/asset/eth/eth_test.go

```diff
@@ -527,3 +528,30 @@ func TestContract(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateSecret(t *testing.T) {
+	secret, blankHash := make([]byte, 32), make([]byte, 32)
+	copy(secret[:], encode.RandomBytes(32))
```

Comment:
I don't care because it's just a test, but no need to reslice `secret` in this copy.